### PR TITLE
API{-proposed}.yaml: Add new style field to SeriesModel

### PR DESCRIPTION
**Pull request by marco-miller**
This change depends on (the reference) o.e.t.incubator trace-server
change [1], below. That change [1]'s commit message explains this fix.

[1] server: Serialize series type in SeriesModelSerializer

Signed-off-by: Marco Miller <marco.miller@ericsson.com>

### CODE REVIEW

File: API-proposed.yaml [hunk 1]
```diff
@@ -2461,16 +2463,21 @@ components:
...
       - title
       type: object
       properties:
         series:
           type: array
+          description: The collection of series
           items:
             $ref: '#/components/schemas/SeriesModel'
+        style:
+          $ref: '#/components/schemas/OutputElementStyle'
```
Comment (bhufmann) [inline API-proposed.yaml:2476]: I think the style element should be in the SeriesModel and not globally across all the series, for example, for series color, symbol or line style. 

We could decide to keep a global style here for common styles and the SeriesModel would have optionally additional or overwriting styles in it.

**Review by bhufmann — CHANGES_REQUESTED** [review]

File: API-proposed.yaml (hunk 1 above)
Comment (marco-miller) [reply API-proposed.yaml:2476]: Ack, and this comment currently applies to the back-end counterpart; will check this.

File: API-proposed.yaml [hunk 2]
```diff
@@ -2209,6 +2209,8 @@ components:
           type: integer
           description: End time for this state
           format: int64
+      description: "If state holds a single applicable value and no style(s), that\
```
Comment (bhufmann) [inline API-proposed.yaml:2212]: This is not related to series model styles and should not be part of this PR.

File: API.yaml [hunk 3]
```diff
@@ -1516,6 +1516,8 @@ components:
           type: integer
           description: End time for this state
           format: int64
+      description: "If state holds a single applicable value and no style(s), that\
+        \ value is this object's parent key"
```
Comment (bhufmann) [inline API.yaml:1520]: unrelated (see above comment)

**Review by bhufmann — CHANGES_REQUESTED** [review]

File: API-proposed.yaml (hunk 2 above)
Comment (marco-miller) [reply API-proposed.yaml:2212]: Indeed, as PR's description above. We already used such PRs and strived for focused (related) commits underneath, as in here. But I can work on splitting it into a few more branches.

Comment (marco-miller) [reply API-proposed.yaml:2212]: Correction, this commit is not focused enough yes, so I'm currently splitting it (this PR) into focused ones. Thanks for the flag.

Comment (marco-miller) [reply API-proposed.yaml]: Done; split into this now focused main PR commit, alongside the new #80 and #81 ones.

File: API.yaml (hunk 3 above)
Comment (marco-miller) [reply API.yaml]: Done, per above.

**Review by bhufmann — COMMENTED** [review]
Looks good to me. You might want to adjust the description for #81.

**Review by bhufmann — COMMENTED** [review]
Looks good to me. Thanks!

**Review by bhufmann — APPROVED** [review]
